lib/media/ver20/get_audio_encoder_configuration.ex

```diff
@@ -1,36 +0,0 @@
-defmodule Onvif.Media.Ver20.GetAudioEncoderConfiguration do
```

Comment:
does this need to be deleted? or can we just fix the namespacing in the request?

Reply:
Yes `GetAudioEncoderConfiguration` doesn't exit in media ver20 onvif spec. This was copy-paste error by me in f4071614b5f0555

Reply:
Tested with uniview since axis doesn't support audio. But axis responded with correct response that audio not supported